src/sonic-config-engine/config_samples.py

```diff
@@ -10,6 +10,16 @@
 else:
     UNICODE_TYPE = unicode
 
+def generate_common_config(data):
+    data['FLEX_COUNTER_TABLE'] = {
```

Comment:
Consider creating another PR with moving PORT, etc. config to generate_common_config